scripts/eia/Rwanda-RAB-AddOn.R

```diff
@@ -155,9 +164,27 @@ carob_script <- function(path) {
    
    d$irrigated <- as.logical(d$irrigated)
    
+   # # Fix irrigation dates
+   
+   d$irrigation_dates <- gsub(" no_answer", "", d$irrigation_dates)
+   d$irrigation_dates <- gsub(" ", "; ", d$irrigation_dates)
+   d$irrigation_dates <- as.character(gsub("jan", "2023-01", d$irrigation_dates))
+   d$irrigation_dates <- gsub("feb", "2023-02", d$irrigation_dates)
+   d$irrigation_dates <- gsub("mar", "2023-03", d$irrigation_dates)
+   d$irrigation_dates <- gsub("apr", "2023-04", d$irrigation_dates)
+   d$irrigation_dates <- gsub("may", "2023-05", d$irrigation_dates)
+   d$irrigation_dates <- gsub("jun", "2023-06", d$irrigation_dates)
+   d$irrigation_dates <- gsub("jul", "2023-07", d$irrigation_dates)
+   d$irrigation_dates <- gsub("aug", "2023-08", d$irrigation_dates)
+   d$irrigation_dates <- gsub("sep", "2023-09", d$irrigation_dates)
+   d$irrigation_dates <- gsub("oct", "2023-10", d$irrigation_dates)
+   d$irrigation_dates <- gsub("nov", "2023-11", d$irrigation_dates)
+   d$irrigation_dates <- gsub("dec", "2022-12", d$irrigation_dates)
+   
    message("yield is given in kg,bags and tonnes instead of kg/ha")
```

Comment:
Coul this be improved? WHat is the average bag weight (maize, potato, rice, etc) in Rwanda?